docs/develop/getting-started/virtualenvs.md

```diff
@@ -57,15 +57,15 @@ Thus in addition to having multiple ``python`` commands, you can have multiple
 A Python virtual environment is *created from a distribution installation*. It
 isolates the Python packages and commands installed.
 
-For instance you have two Python programs A and B. Python program A depends on
+For instance, if you have two Python programs A and B, and Python program A depends on
 version 1 of package X, and program B depends on version 2 of the same
 package X. Both version 1 and version 2 cannot be installed at the same time
 in a single Python distribution.
 
-Virtual environments allows you to have a virtual environment for program A and
-another for program B each with different version installed of package X.
+Virtual environments allow you to have a virtual environment for program A and
+another for program B each with a different version installed of package X.
 
-This means that a virtual environment has it's own ``python`` and ``pip``
+This means that a virtual environment has its own ``python`` and ``pip``
 commands that are *enhanced aliases* to the distributions ``python`` and ``pip``
```

Comment:
Don't know too much about that *enhanced aliases* business to be honest... I think cutting it at:

```
This means that a virtual environment has its own ``python`` and ``pip`` commands.
```

is good enough.